dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -470,6 +471,17 @@
 			status = "disabled";
 			label = "DMA_2";
 		};
+
+                quadspi: quadspi@a0001000 {
+                        compatible = "st,stm32-qspi";
+                        #address-cells = <0x1>;
+                        #size-cells = <0x0>;
+                        reg = <0xa0001000 0x34>;
```

Comment:
usually, the reg bank for the quadspi is larger ie 0x400 at 0xa0001000 even if strictly 0x34-bytes are available for the registers

Reply:
This was copied directly from dts/arm/st/f7/stm32f7.dtsi - line last modified in 30dbc409095 
Happy to extend it to cover the whole reg bank though.